core/src/main/java/com/xmartlabs/base/core/helper/ui/UiHelper.java

```diff
@@ -147,7 +148,8 @@ public static void validateFieldsAndPutErrorMessages(List<EditText> fields, bool
    * @param hideErrorLayout whether or not to enable the error on the given {@link EditText}
    * @return true if {@code editText} is valid using {@link #isValidField(EditText)}
    */
-  public static boolean validateField(EditText editText, boolean hideErrorLayout) {
+  public static boolean validateField(@NonNull EditText editText, boolean hideErrorLayout,
```

Comment:
This doesn't seem to be enough generic as it names would indicate. This function takes two kind of error messages: a) for invalid emails and b) for required fields; so I guess this should be used just in that cases (for an email text field or a required one). What should be used in other case? For example, validating if a *full name* field contains a first and last name?